### DISCUSSION TITLE
Mac OS

### DISCUSSION THREAD

**Discussion by CyrilleHome** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#118] [Q&A] [answered]
it is works for Mac OS ? if yes how ? thx !

**Comment by PiotrMachowski** [comment, marked as answer]
You can use the [python version](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor?tab=readme-ov-file#manual-run-in-python)

**Reply by CyrilleHome** [reply]
it works ! many thanks !
